jsonapi/jsonapi_utility.go

```diff
@@ -139,8 +140,11 @@ type Conflict interface {
 
 // JSONErrorResponse auto maps the provided error to the correct response type
 // If all else fails, InternalServerError is returned
-func JSONErrorResponse(x InternalServerError, err error) error {
-	jsonErr, status := ErrorToJSONAPIErrors(err)
+func JSONErrorResponse(obj interface{}, err error) error {
+	x := obj.(InternalServerError)
+	c := obj.(context.Context)
```

Comment:
Understood, InternalServerError can represent a Context ( because of the interface definition )